hypothesis-python/src/hypothesis/extra/pytestplugin.py

```diff
@@ -65,9 +65,18 @@ def pytest_addoption(parser):
     )
 
 
+def pytest_report_header(config):
+    profile = config.getoption(LOAD_PROFILE_OPTION)
+    if not profile:
+        profile = 'default'
+    settings_str = settings.get_profile(profile).show_changed()
+    if settings_str != '':
+        settings_str = ' -> %s' % (settings_str)
+    return 'hypothesis profile %s%s' % (profile, settings_str)
```

Comment:
Needs to be `%r%s` here, because (while deprecated) it is still possible to have different profiles registered for the integer `5` and the string `'5'` :sob:

Reply:
ack